Dockerfile

```diff
@@ -1,33 +1,39 @@
 ARG base_version=1.2.0
 ARG registry=quay.io
+ARG NAME=indexd
```

Comment:
`NAME` is defined on both line 3 and line 6, I just want to confirm that's intentional.

Reply:
only define it in line 3 (globally) does not work in multi-stage build for me. 
Have to define it again to be able to used in stages. 
That's my understanding. 
Please let me know if I was wrong.

Reply:
I think you are more of an expert in docker related matters than I am so I trust your judgement. I just wanted to make sure it was intentional. Thank you!